WireAPI/Sources/WireAPI/APIs/VerificationCodeAPI/VerificationCodeAPI.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+// sourcery: AutoMockable
+/// An API access object for endpoints concerning verification codes.
+public protocol VerificationCodeAPI {
+
+    /// Request a verification code for a provided email address.
+    ///
+    /// - Parameter email: Email address of the account
+    /// - Returns: The user details.
+
+    func requestVerificationCode(for email: String) async throws
```

Comment:
question: since the verification code is part of authentication, what do you think about moving this method to the `AuthenticationAPI`?